protocol/writing-resource/slash-semantics-exclude.feature

```diff
@@ -47,6 +48,7 @@ Feature: With and without trailing slash cannot co-exist
     Given url childContainerUrl
     And headers clients.alice.getAuthHeaders('PUT', childContainerUrl)
     And header Content-Type = 'text/turtle'
+    And header Link = '<http://www.w3.org/ns/ldp#BasicContainer>; rel="type"'
```

Comment:
Same as https://github.com/solid-contrib/specification-tests/pull/77/files#r888847236

Reply:
I hadn't spotted that PUT with "/" just doesn't need the link since it is implied. I will remove that but I'll add equivalent tests for POST.